bibliotheque-universitaire-de-medecine-vancouver.csl

```diff
@@ -46,6 +46,31 @@
       </term>
     </terms>
   </locale>
+  <locale xml:lang="en">
+    <date form="text" delimiter=" ">
+      <date-part name="year"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="day"/>
+    </date>
+    <terms>
+      <term name="retrieved">available from</term>
+      <term name="in">in</term>
+      <term name="cited">cited</term>
+      <term name="internet">Internet</term>
+      <term name="collection-editor" form="long">
+        <single>editor</single>
+        <multiple>editors</multiple>
+      </term>
+      <term name="editor">
+        <single>editor</single>
+        <multiple>editors</multiple>
+      </term>
+      <term name="original-author">
```

Comment:
This seems left over from the French? And/or should it be `inventor`?

Reply:
I removed it in both languages. It was unnecessary for this style and probably a left over from the template style.